django/db/migrations/writer.py

```diff
@@ -132,8 +132,9 @@ class MigrationWriter:
     of the migration file from it.
     """
 
-    def __init__(self, migration):
+    def __init__(self, migration, no_header=False):
         self.migration = migration
+        self.no_header = no_header
```

Comment:
It feels like `MigrationWriter` shouldn't have to deal with a negated option and it should be the commands responsibility to deal with the fact the flag is negated. That's the main reason why I suggested `include_header=True` in the previous PR.